bcipy/gui/file_dialog.py

```diff
@@ -25,18 +27,21 @@ def __init__(self):
 
         # The native dialog may prevent the selection from closing after a
         # directory is selected.
-        self.options = QFileDialog.Option(3)
+        self.options = QFileDialog.Option.DontUseNativeDialog
```

Comment:
Note that this was previously broken because the Option enum changed between pyqt5 and 6.